src/uwtools/resources/FV3Forecast.jsonschema

```diff
@@ -90,41 +221,54 @@
           "type": "string"
         }
       },
+      "required": [
+        "scheduler"
+      ],
       "type": "object"
     },
     "preprocessing": {
+      "additionalProperties": false,
       "properties": {
         "lateral_boundary_conditions": {
+          "additionalProperties": false,
           "properties": {
             "interval_hours": {
-              "default": 3,
               "minimum": 1,
-              "type": "number"
+              "type": "integer"
             },
             "offset": {
-              "default": 0,
               "minimum": 0,
-              "type": "number"
+              "type": "integer"
             },
             "output_file_path": {
-              "format": "uri",
               "type": "string"
             }
           },
+          "required": [
+            "interval_hours",
+            "offset",
+            "output_file_path"
+          ],
           "type": "object"
         }
       },
+      "required": [
+        "lateral_boundary_conditions"
+      ],
```

Comment:
This would only be required for a regional run. I'm not sure if we want to worry about that just yet since the whole thing is pretty hard-coded for that case at the moment.

Reply:
I _think_ this is currently mandatory from the code's perspective: `_define_boundary_files()` has
```
lbcs_config = self._experiment_config["preprocessing"]["lateral_boundary_conditions"]
```

and that method is unconditionally called by `prepare_directories()`, which is called by both `_run_via_batch_submission()` and `_run_via_local_execution()`, _one of which_ is called by `run()`, all in class `FV3Forecast`. So, I think the code will crash, given the current control flow, if this key is not present.

Reply:
Let's leave it as-is for now and remember that we have work to do to move to supporting global configurations.